src/routes/clients.js

```diff
@@ -1,9 +1,15 @@
 const router = require('express').Router();
 const Joi = require('joi');
+Joi.objectId = require('joi-objectid')(Joi);
 const wrapper = require('../middleware/wrapper');
 const validator = require('../middleware/validator');
 
-const { createClient } = require('../controllers/clients');
+const { getByID, createClient } = require('../controllers/clients');
+
+const validatorGetById = Joi.object({

```

Comment:
Can we call this validator `validatorClientID` and use it for PUT and DELETE endpoints in future?

Reply:
We can if u rename it to "validatorClientID" before using it.